vortex-datafusion/src/persistent/statistics.rs

```diff
@@ -0,0 +1,42 @@
+use arrow_array::cast::AsArray;
+use arrow_array::types::UInt64Type;
+use datafusion::functions_aggregate::min_max::{MaxAccumulator, MinAccumulator};
+use datafusion_common::stats::Precision;
+use datafusion_common::ColumnStatistics;
+use datafusion_expr::Accumulator;
+use vortex_array::array::StructArray;
+use vortex_array::variants::StructArrayTrait as _;
+use vortex_array::IntoCanonical;
+use vortex_error::VortexResult;
+
+pub fn array_to_col_statistics(array: StructArray) -> VortexResult<ColumnStatistics> {
```

Comment:
Columns have `UncompressedSizeInBytes` stat, we could use it to report `SizeInBytes` to datafusion